website/api_versioned_docs/version-1.1/blockchain-integration/contracts/verifier.md

```diff
@@ -31,38 +31,217 @@ In this example, the `IS_EVEN` zkVM program verifies that the number, `x`, is ev
+| [RiscZeroVerifierEmergencyStop][estop-src] | [`0x5E36f0D56741013d864d8FEb5950AB0E7Eff9Ab1`][estop-10-etherscan]    |
+| TimelockController                         | [`0xdc986a09728f76110ff666ee7b20d99086501d15`][timelock-10-etherscan] |
+
+### Optimism Sepolia (11110)
+
+| Contract                                   | Address                                                                  |
+| ------------------------------------------ | ------------------------------------------------------------------------ |
+| [RiscZeroVerifierRouterRouter][router-src] | [`0xB369b4dd27FBfb59921d3A4a3D23AC2fc32FB908`][router-11110-etherscan]   |
+| [RiscZeroGroth16Verifier][verifier-src]    | [`0x84b943E31e7fAe6072ce5F75eb4694C7D5F9b0cF`][verifier-11110-etherscan] |
+| [RiscZeroVerifierEmergencyStop][estop-src] | [`0x5E36f0D56741013d864d8FEb5950AB0E7Eff9Ab1`][estop-11110-etherscan]    |
+| TimelockController                         | [`0x2DEfEA335392bb62d01f74e338697C7B31De254C`][timelock-11110-etherscan] |
+
+### Linea Mainnet (59144)
+
+| Contract                                   | Address                                                                  |
+| ------------------------------------------ | ------------------------------------------------------------------------ |
+| [RiscZeroVerifierRouterRouter][router-src] | [`0x0b144e07a0826182b6b59788c34b32bfa86fb711`][router-59144-etherscan]   |
+| [RiscZeroGroth16Verifier][verifier-src]    | [`0x84b943E31e7fAe6072ce5F75eb4694C7D5F9b0cF`][verifier-59144-etherscan] |
+| [RiscZeroVerifierEmergencyStop][estop-src] | [`0x5E36f0D56741013d864d8FEb5950AB0E7Eff9Ab1`][estop-59144-etherscan]    |
+| TimelockController                         | [`0xdc986a09728f76110ff666ee7b20d99086501d15`][timelock-59144-etherscan] |
+
+### Polygon zkEVM Mainnet (1101)
+
+| Contract                                   | Address                                                                 |
+| ------------------------------------------ | ----------------------------------------------------------------------- |
+| [RiscZeroVerifierRouterRouter][router-src] | [`0x0b144e07a0826182b6b59788c34b32bfa86fb711`][router-1101-etherscan]   |
+| [RiscZeroGroth16Verifier][verifier-src]    | [`0x84b943E31e7fAe6072ce5F75eb4694C7D5F9b0cF`][verifier-1101-etherscan] |
+| [RiscZeroVerifierEmergencyStop][estop-src] | [`0x5E36f0D56741013d864d8FEb5950AB0E7Eff9Ab1`][estop-1101-etherscan]    |
+| TimelockController                         | [`0xdc986a09728f76110ff666ee7b20d99086501d15`][timelock-1101-etherscan] |
+
+<!-- GENERATED CONTENT END-->
 
 <!-- TODO: Move this example into risc0-ethereum such that it will be under the same version management -->
 
-[doc-released-contracts]: /api/blockchain-integration/contracts/verifier#contract-addresses
+[deployment.toml]: https://github.com/risc0/risc0-ethereum/blob/release-1.1/contracts/deployment.toml
```

Comment:
Autogenerated links were placed down here by the formatter. We could do a it of work to prevent this, but I think it's ok.